src/app/components/SubjectHeading/SubjectHeadingPageWrapper.jsx

```diff
@@ -26,31 +27,35 @@ const SubjectHeadingPageWrapper = (props) => {
   const [label, setLabel] = useState('');
 
   return (
-    <div>
-      <div className="subjectHeadingsBanner">
-        <div className="subjectHeadingBannerNav">
-          <Breadcrumbs type="subjectHeading" headingDetails={!!subjectHeadingUuid}/>
-          <SubjectHeadingSearch />
+    <DocumentTitle title="Subject Headings">
```

Comment:
I think if these extra divs are necessary to match SCC currently, that’s fine. Are you sure DocumentTitle is necessary for the frontend? It does rely on an unsafe lifecycle method. So ideally, we would figure out a different solution. 
Seems like we could create a function to pass the page content into so that we don’t need to repeat this in many different components.